migrations/Version20180424124131.php

```diff
@@ -2,15 +2,16 @@
 
 namespace DoctrineMigrations;
 
-use Doctrine\DBAL\Migrations\AbstractMigration;
+
 use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
 
 /**
  * Auto-generated Migration: Please modify to your needs!
  */
 class Version20180424124131 extends AbstractMigration
 {
-    public function up(Schema $schema)
+    public function up(Schema $schema):void
```

Comment:
I suppose our coding standards don't check migration files, but if we change old migration files we should probably modify them to be inline with how newly generated ones look. That is a space in between the colon and the return type.

Reply:
Our coding standards should check all code we write, i.e. also the migrations.